tests/unit/backend/layers/business/test_business.py

```diff
@@ -0,0 +1,802 @@
+from datetime import datetime
+import unittest
+import copy
+from unittest.mock import Mock
+from backend.corpora.common.providers.crossref_provider import CrossrefException, CrossrefProvider
+from backend.layers.business.business import BusinessLogic, CollectionQueryFilter, UserInfo
+from backend.layers.common.entities import CollectionMetadata, CollectionVersion, DatasetMetadata, DatasetStatus, Link
+from backend.layers.persistence.persistence import DatabaseProviderInterface
+from backend.layers.thirdparty.crossref_provider import CrossrefProviderInterface
+from backend.layers.thirdparty.step_function_provider import StepFunctionProviderInterface
+from tests.unit.backend.layers.persistence.persistence_mock import DatabaseProviderMock
+
+class BaseBusinessLogicTestCase(unittest.TestCase):
+
+    sample_collection_metadata = CollectionMetadata(
+        "test collection 1", 
+        "description of test collection 1",
+        "test_user_1",
+        "scientist",
+        "scientist@czi.com",
+        []
+    )
+
+    sample_dataset_metadata = DatasetMetadata(
+        "test_organism",
+        "test_tissue",
+        "test_assay",
+        "test_disease", 
+        "test_sex",
+        "test_self_reported_ethnicity",
+        "test_development_stage",
+        "test_cell_type",
+        10
+    )
+
+    test_user_name = "test_user_1"
+
+    user_info = UserInfo(
+        "test_user_1",
+        "fake token"
+    )
+
+    def setUp(self) -> None:
+        self.database_provider = DatabaseProviderInterface() # replace with DatabaseProviderMock()
+
+        # By default does nothing. Can be mocked by single test cases.
+        self.crossref_provider = CrossrefProviderInterface()
+        self.step_function_provider = StepFunctionProviderInterface()
+
+        self.business_logic = BusinessLogic(
+            database_provider=self.database_provider, 
+            crossref_provider=self.crossref_provider,
+            step_function_provider=self.step_function_provider,
+        )
+        return super().setUp()
```

Comment:
Remove. You don't need to call super or return it.

Reply:
Sounds good, this was recommended by VSCode, but I see it isn't needed.